<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectsTab</class>
 <widget class="QWidget" name="ProjectsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="3" rowspan="4">
    <widget class="QGroupBox" name="projectGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Project: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0,0,0">
      <item row="1" column="1">
       <widget class="QLineEdit" name="smmRootLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="russellLabel">
        <property name="text">
         <string>Russell root</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="smmLabel">
        <property name="text">
         <string>SMM root</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="russellRootLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="chooseRusRootButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>choose</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="chooseSmmRootButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>choose</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="addProjectButton">
     <property name="text">
      <string>add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="projectsComboBox">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="delProjectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>del</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="projetcsLabel">
     <property name="text">
      <string>Projects:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>191</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
